src/Umbraco.Infrastructure/Packaging/PackageDataInstallation.cs

```diff
@@ -813,27 +807,40 @@ private T CreateContentType<T>(Guid key, T parent, int parentId, string alias)
             }
         }
 
-        private void UpdateContentTypesTabs<T>(T contentType, XElement tabElement)
+        private void UpdateContentTypesPropertyGroups<T>(T contentType, XElement propertyGroupsContainer)
             where T : IContentTypeComposition
         {
-            if (tabElement == null)
+            if (propertyGroupsContainer == null)
                 return;
 
-            var tabs = tabElement.Elements("Tab");
-            foreach (var tab in tabs)
+            var propertyGroupElements = propertyGroupsContainer.Elements("Tab");
+            foreach (var propertyGroupElement in propertyGroupElements)
             {
-                var caption = tab.Element("Caption").Value;
+                var name = propertyGroupElement.Element("Caption").Value; // TODO Rename to Name (same in EntityXmlSerializer)
```

Comment:
That would be a big breaking change in the package XML/UDT export files, so if we're doing this now, I would recommend also updating:
- `Tabs`/`Tab` into `PropertyGroups`/`PropertyGroup` and remove the `Id` element (containing the numeric identifier that's not used anymore, but couldn't be removed because the import logic required it to exist)
- `GenericProperties`/`GenericProperty` into `PropertyTypes`/`PropertyType`

Reply:
It can wait :P But maybe we should have it on a list for v10 or v11..